### ISSUE TITLE
Profile.getMemberUrlResources() returns a single Resources

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#30]
```
Hi Nabeel,

thanks for the work you've done. Could you please have a look at this issue ?

What steps will reproduce the problem?
1. Authenticate
2. Get a User object with 
getProfileForCurrentUser(EnumSet.of(ProfileField.MEMBER_URL_RESOURCES));
3. Call profile.getMemberUrlResources();

What is the expected result? What do you see instead?
Expected a MemberUrlResources instance that gives access to multiple resources
Received a MemberUrlResources instance with a single .getMemberUrl() method

As long as it is possible to enter multiple websites on a profile I expected to 
get multiple 
resources.

Thanks

Nicolas
```

Original issue reported on code.google.com by `nicolas....@gmail.com` on 4 Jun 2010 at 2:03

**Comment by GoogleCodeExporter** [comment]
```
Thanks for pointing this out. It will be fixed in the next build on Monday.
```

Original comment by `nabeelmukhtar` on 11 Jun 2010 at 7:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi Nabeel,

thanks for sorting that out. I've checked the trunk sources and do not see any 
changes in the MemberUrlResources class. Is the fix available in a branch or am 
I missing something ?

Nicolas
```

Original comment by `nicolas....@gmail.com` on 1 Jul 2010 at 7:04

**Comment by GoogleCodeExporter** [comment]
```
This defect has been fixed in build linkedin-j-1.0-rc1.zip.
```

Original comment by `nabeelmukhtar` on 9 Aug 2010 at 10:42
- Changed state: **Fixed**
